src/drive/web/modules/upload/index.js

```diff
@@ -236,6 +240,32 @@ export const overwriteFile = async (client, file, path) => {
   return resp.data
 }
 
+export const uploadFilesFromNative = (
+  files,
+  folderId,
+  uploadFilesSuccessCallback
+) => async dispatch => {
+  dispatch({
+    type: ADD_TO_UPLOAD_QUEUE,
+    files: files
+  })
+  //!TODO Promise.All
```

Comment:
Any reason not to do it now?

Reply:
I didn't produce any test with 10 concurrent uploads. I don't how it'll react on a slow or shitty network. 

I'll add a note about it in the todo. I think this should be done when working on our upload system one day ;)